include/cupla/types.hpp

```diff
@@ -23,9 +23,10 @@
 
 #include <alpaka/alpaka.hpp>
 #include <cstdint>
-
+#include "cupla/defines.hpp"
 #include "cupla/namespace.hpp"
 
+
```

Comment:
Should I remove the lines below (30-104), if you move them to `defines.hpp` ?

Reply:
ohh yes please remove it. I thought I did it. I wonder why I get no warning that the defines are already defined